python/setup.py

```diff
@@ -245,6 +252,8 @@ def get_thirdparty_packages(packages: list):
 
 
 def download_and_copy(name, src_path, variable, version, url_func):
+    if os.environ.get("TRITON_OFFLINE_BUILD", False):
+        return
```

Comment:
Can you already do that setting `variable`?

Reply:
I didn't quite catch that, wdym by `variable`?

Reply:
I meant in this function:
```
def download_and_copy(name, src_path, variable, version, url_func):
```
there is a `variable` for each binary we download but I think you answered it in the other discussion